### ISSUE TITLE
Feature request: Download abstract

### ISSUE THREAD

**Issue by VagusVet** [RoadToDream/ZotMeta#3]
Any chance of being able to download / refresh the abstract?

**Comment by RoadToDream** [comment]
> Any chance of being able to download / refresh the abstract?

Yes. But abstract from doi general doesn't have good quality, that's why it's not added at the first place

**Comment by RoadToDream** [comment]
Added in v2.0. Thanks for the info.
